<launch>

  <!-- launch the experiment controller node -->
  <node name="experiment_controller" pkg="bwi_exp1" type="experiment.py">
    <param name="graph_file" value="$(find bwi_exp1)/maps/graph2.yaml" />
    <param name="experiment_file" value="$(find bwi_exp1)/config/experiment2.yaml" />
    <param name="control_experiments_file" value="$(find bwi_exp1)/config/exp2_pair1.yaml" />
    <param name="test_experiments_file" value="$(find bwi_exp1)/config/exp2_pair2.yaml" />
    <param name="tutorials_file" value="$(find bwi_exp1)/config/exp2_tutorials.yaml" />
    <param name="data_dir" value="$(find bwi_exp1)/data" />
  </node>

  <node name="robot_positioner" pkg="bwi_exp1" type="robot_positioner">
    <param name="map_file" value="$(find bwi_exp1)/maps/exp1.yaml" />
    <param name="graph_file" value="$(find bwi_exp1)/maps/graph2.yaml" />
    <param name="robot_radius" value="0.25" />
    <param name="robot_padding" value="0.1" />
    <!-- <param name="debug" value="true" /> -->
  </node>

</launch>
